.gitignore

```diff
@@ -5,6 +5,7 @@ __pycache__
 .DS_Store
 /dist
 /build
+/wheelhouse
```

Comment:
Also, I added this since it gets generated if you run cibuildwheel locally, which I did when testing a PyPy wheel:

```bash
pipx run cibuildwheel --only pp310-manylinux_x86_64
```